dotCMS/src/main/java/com/dotmarketing/portlets/htmlpageviews/action/ViewHTMLPageViewsAction.java

```diff
@@ -14,13 +14,10 @@
 import com.dotmarketing.business.APILocator;
 import com.dotmarketing.business.web.HostWebAPI;
 import com.dotmarketing.business.web.WebAPILocator;
-import com.dotmarketing.cache.VirtualLinksCache;
 import com.dotmarketing.exception.DotHibernateException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.exception.DotHibernateException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)